vast.py

```diff
@@ -1091,17 +1157,60 @@ def cloud__copy(args: argparse.Namespace):
     if (args.explain):
         print("request json: ")
         print(req_json)
-    
+        
     r = http_post(args, url, headers=headers,json=req_json)
     r.raise_for_status()
     if (r.status_code == 200):
         print("Cloud Copy Started - check instance status bar for progress updates (~30 seconds delayed).")
         print("When the operation is finished you should see 'Cloud Cody Operation Finished' in the instance status bar.")  
+        if (args.schedule):
+            cli_command = "cloud copy"
+            api_endpoint = "/api/v0" + "/commands/rclone/"
+            add_scheduled_job(args, req_json, cli_command, api_endpoint, "POST") 
     else:
         print(r.text);
         print("failed with error {r.status_code}".format(**locals()));
 
 
+
+
+def add_scheduled_job(args, req_json, cli_command, api_endpoint, request_method):
+    start_time, end_time, time_interval = validate_schedule_values(args)
+
+        
+    schedule_job_url = apiurl(args, f"/commands/schedule_job/")
+    start_date = millis_to_date(start_time)
+    end_date = millis_to_date(end_time)
+    request_body = {
+                "start_time": start_time, 
+                "end_time": end_time, 
+                "time_interval": time_interval,  
+                "api_endpoint": api_endpoint,
+                "request_method": request_method,
+                "request_body": req_json
+            }
+                # Send a POST request
+    response = requests.post(schedule_job_url, headers=headers, json=request_body)
```

Comment:
why ask for request_method if only POST requests are called?

Reply:
scheduled_jobs table is intended to store GET, POST, PUT, or DELETE requests. but call to scheduled_jobs api endpoint will be post if inserting a new record and PUT if updating